5179 cron tab visualisation (#5181)
platformsh/platformsh-docs: sites/upsun/src/increase-observability/application-metrics/cron-metrics.md

---
title: Monitor Cron job executions
description: Learn how to visualize and interpret your Cron job runs directly in the console.

> weight: -50

Author: ```suggestion
weight: -5
```

keywords:
  - "cron"
  - "jobs"
  - "monitor"
  - "logs"
  - "console"
---

You can view your [Cron job](/create-apps/app-reference/single-runtime-image.html#crons) executions directly from your project’s Services > Apps or Services > Workers pages. This visualization helps you quickly understand how scheduled jobs are performing and spot unusual patterns without manually checking logs.

Each execution is represented visually, letting you monitor performance trends and failures at a glance.

This makes it easier to:

- Verify that scheduled jobs are running on time.
- Identify failed or unusually long-running jobs.
- Drill into logs only when something looks wrong.

## How to view Cron job executions

1. Open your project in the Console.  

> 2. Navigate to:

Author: Added an extra step for this, thank you! :)

>    - **Services > Apps** or  

Reviewer: Apps and Workers are both under **Apps**

<img width="391" height="367" alt="image" src="https://github.com/user-attachments/assets/219dde53-a652-441e-abc7-22862c93db4f" />

Reviewer: so 
Services --> Apps & Services --> Apps --> app
or
Services --> Apps & Services --> Apps --> Workers

   - **Services > Workers** (for background jobs).  
3. Click the **Crons** tab to view your scheduled jobs.  

Each Cron job displays a visual history of its executions over time.

## Understanding the visualization

Each execution appears as a vertical bar in the visualization:

| Visual element | Meaning | Example |
|----------------|----------|----------|

> | **Color** | Indicates success or failure. | Green = success, Red = failure |

Reviewer: can we add some other signifier to failures that *isnt* color to make them stand out? maybe wider? outlined?

> | **Color** | Indicates success or failure. | Green = success, Red = failure |

Reviewer 2: Hi @gilzow. This is a question for @dudenhofer. Aaron, could we iterate on this basis to help colorblind users better differentiate successes and failures? Texture maybe?

Let's document the feature on its the current status, then, update the docs once an improved version is released.
Does this sound good?

| **Height** | Proportional to the job’s execution time. | Taller bars indicate longer runtimes |
| **Click a bar** | Opens detailed logs for that specific run. | Review stdout/stderr for debugging |

> This combination of color and height makes it easy to spot outliers, for example:

Author: ```suggestion
This combination of color and height helps in spotting outliers, for example:
```
